_posts/2017-10-02-Testing-angular-with-karma.md

```diff
@@ -0,0 +1,513 @@
+---
+layout: post
+authors: [martijn_willekens]
+title: "Testing Angular with Karma 101"
+image: /img/2017-10-02-testing-angular-with-karma/unit-tests.png
+tags: [Angular,Karma,Unit testing,Jasmine,TypeScript]
+category: Angular
+comments: true
+---
+Testing your code is as important as the writing code itself. 
+This also counts for frontend applications such as Angular apps. 
+Unit testing is one way to do so. 
+The goal of these kind of tests is to isolate classes and verify the output of its functions to be what you expect when they are called.
+We also need a tool to run our tests written in TypeScript.
+[Karma](https://karma-runner.github.io/) is such a tool and can be used to run tests as described in this blog.
```

Comment:
You would want to open links in a new tab so that our readers remain on out tech blog:
[Foo](http://google.be){:target="_blank"}